content/docs/intro/vs/cloud_templates.md

```diff
@@ -1,7 +1,6 @@
 ---
-title: Pulumi vs. Cloud Templates (AWS CloudFormation, etc.)
-meta_desc: This page gives an overview of the major differences between Pulumi
-           and Cloud Templates (AWS CloudFormation, etc.).
+title: Pulumi vs. AWS CloudFormation and Other Templates | Pulumi
```

Comment:
A while back, we used to auto-append "| Pulumi" to all page titles. Is that worth bringing back?

Reply:
There's room for adjusting to suit your brand, as it's not a super important SEO issue, but Moz does recommend the "| Brand Name" format for title tags in Google; it looks good and keeps things uniform, so that's what we typically recommend.

Reply:
In most cases (including this page), we do indeed append this automatically, yep:

![image](https://user-images.githubusercontent.com/274700/73801368-122b7000-476f-11ea-9686-c154b522bb38.png)

https://github.com/pulumi/docs/blob/master/layouts/partials/head.html#L66-L71

(So there's no need to specify these manually in frontmatter titles, @97thFloor.)